orga/app/components/organization-credit-info.js

```diff
@@ -0,0 +1,26 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { htmlSafe } from '@ember/string';
+
+export default class OrganizationCreditInfoComponent extends Component {
+    @service currentUser
+
+    get tooltipContent() {
+      return htmlSafe('Le nombre de crédits affichés correspond au nombre de crédits acquis par l’organisation et en cours de validité (indépendamment de leur activation). Pour plus d’information contactez-nous à l’adresse <a href=mailto:pro@pix.fr>pro@pix.fr</a>');
+    }
+    get credit() {
+      return this.currentUser.organization.credit.toLocaleString();
+    }
+
+    get hasRight() {
+      return this.currentUser.isAdminInOrganization;
+    }
+
+    get creditIsMoreThanZero() {
+      return this.currentUser.organization.credit > 0;
+    }
+
+    get showCredit() {
```

Comment:
```suggestion
    get canShowCredit() {
```
mettre condition hasRight et creditIsMoreThanZero